src/components/movieContainer/index.tsx

```diff
@@ -20,6 +20,7 @@ export class MoviesContainer extends React.Component<any, any> {
 }
 
 const mapStateToProps = (state: any) => ({
+    //error here

```

Comment:
try this:
const mapStateToProps = (state: any) => {
// now here console log the state and check f you are getting somthing
  return {
movies: state.movies.movies
}
}